index.js

```diff
@@ -205,7 +208,7 @@ module.exports = function (opts) {
         filename: false,
         style: 'solarized-dark',
         template: ROOT+'src/layout.html',
-        scss: ROOT+'src/app.scss',
```

Comment:
My intention was to move where it was to move it not remove it - I have just done another commit to this pull request which brings it inline with how variables are used.

BUT the extra scss needs looking at as a layout I had working yesterday breaks when I add your CSS into the mix.